### PR TITLE
added web-server firewall to Manual

### PR DESCRIPTION
**Pull request by cjakeman**
Forum www.elvastower.com/forums/index.php?/topic/36085-windows-firewall
Now taht the web server is always available, the Manual needs a mention of setting the Windows Firewall to allow communication.

### CODE REVIEW

File: Source/Documentation/Manual/software-platform.rst [hunk 1]
```diff
@@ -101,15 +102,26 @@ the software will contribute to better overall performance -- potentially
...
+
+The simplest way to access these samples is to start the game and then launch
+a browser on the smae PC. Then enter "localhost:2150" into your browser address bar. 
+(2150 is the default port number set in :ref:`Menu > Options > General <options-web-server-port>`)
+
 Sample Web Pages
 ----------------
 
-These pages give examples of what can be done with the APIs.
+A number of web-pages are included in the Open Rails isntallation as examples of
```
Comment (cesarBLG) [inline Source/Documentation/Manual/software-platform.rst:123]:
```suggestion
A number of web-pages are included in the Open Rails installation as examples of 
```

**Review by cesarBLG — CHANGES_REQUESTED** [review]
Hi Chris,
Seems OK, just one typo to correct.

**Comment by cjakeman** [comment]
Hi Cesar,

Thanks for spotting this.

Hope you can approve this now.

Best wishes,

Chris

File: Source/Documentation/Manual/software-platform.rst [hunk 2]
```diff
@@ -101,15 +102,26 @@ the software will contribute to better overall performance -- potentially
...
 ==========
 
 .. _sample-web-pages:
 
+The game uses a built-in web-server to deliver standard and custom web-pages
+to any browser. This can be running on the same PC as Open Rails or another PC 
+or other device which is connected to your local network.
+
+The simplest way to access these samples is to start the game and then launch
+a browser on the smae PC. Then enter "localhost:2150" into your browser address bar.
```
Comment (cesarBLG) [inline Source/Documentation/Manual/software-platform.rst:117]:
```suggestion
a browser on the same PC. Then enter "localhost:2150" into your browser address bar. 
```
I've just found another typo. Sorry for not having spotted this previously

Comment (cjakeman) [reply Source/Documentation/Manual/software-platform.rst:117]: I've noticed that my typing is getting less reliable as I am getting older.
Thanks for spotting this.

**Review by cesarBLG — APPROVED** [review]
Thanks Chris, I think it is OK now.

**Pull request merged**